lib/jxl/enc_file.cc

```diff
@@ -58,8 +58,6 @@ PassDefinition progressive_passes_dc_lf_salient_ac_other_ac[] = {
      /*suitable_for_downsampling_of_at_least=*/0}};
 
 PassDefinition progressive_passes_dc_quant_ac_full_ac[] = {
-    {/*num_coefficients=*/8, /*shift=*/2, /*salient_only=*/false,
-     /*suitable_for_downsampling_of_at_least=*/4},
     {/*num_coefficients=*/8, /*shift=*/1, /*salient_only=*/false,
      /*suitable_for_downsampling_of_at_least=*/2},
```

Comment:
reflecting the LCP experiments by Moritz there wouldn't be any downsampling here -- just a shift by 1

what is the impact of 'suitable for downsampling of at least' ?

Reply:
There's no downsampling done in encoder or decoder here, that field is just a decoder hint that gets signalled in the headers, basically saying that if after decoding, you're going to downsample it 1:2  (e.g. because it's a 1000x1000 image that will be rendered in a screen area of 500x500), then maybe just decoding up to this pass will be 'good enough'.